caput/containers/__init__.py

```diff
@@ -1,13 +1,62 @@
-"""caput.containers.
+"""A high-level in-memory data container format for :py:mod:`caput.pipeline`.
 
-A high-level in-memory data container format for `caput.pipeline`.
+Containers are built on top of :py:class:`~caput.memdata.BasicCont` and provide a structured way
+to hold multi-dimensional data arrays with named axes and dataset specifications.
+New containers should generally inherit from :py:class:`.ContainerBase` or
+:py:class:`.TableBase`.
 
-Submodules
-----------
-.. autosummary::
-   :toctree: _autosummary
+Examples
+--------
+Containers are defined by subclassing :py:class:`ContainerBase` and specifying
+the axes and dataset specifications:
```

Comment:
```suggestion
the axes and datasets:
```
To avoid saying "specify/specification" twice.